lib/support/cli.js

```diff
@@ -273,6 +273,9 @@ module.exports.parseCommandLine = function parseCommandLine() {
     .option('preURL', {
       describe: 'A URL that will be accessed first by the browser before the URL that you wanna analyze. Use it to fill the cache.'
     })
+    .option('userTimingWhitelist', {
+      describe: 'A regex that will whitelist which userTimings to capture in results.'
```

Comment:
I think we could just add what happens per default if we leave it out so that is clear.